toplevel/byte/topmain.ml

```diff
@@ -156,19 +156,25 @@ let prepare ppf =
       Format.fprintf ppf "Uncaught exception: %s\n" (Printexc.to_string x);
       false
 
-(* If [name] is "", then the "file" is stdin treated as a script file. *)
 let file_argument name =
```

Comment:
Could this be renamed into `input_argument`, with `file_argument` being an alias for the previous behavior which you could use in `parse_arguments` for example?